staging/src/k8s.io/kubectl/pkg/drain/drain.go

```diff
@@ -315,7 +315,7 @@ func (d *Helper) evictPods(pods []corev1.Pod, evictionGroupVersion schema.GroupV
 				} else if apierrors.IsForbidden(err) && apierrors.HasStatusCause(err, corev1.NamespaceTerminatingCause) {
 					// an eviction request in a deleting namespace will throw a forbidden error,
 					// if the pod is not marked deleted, we retry until it is.
-					fmt.Fprintf(d.ErrOut, "error when evicting pod %q (will retry after 5s): %v\n", activePod.Name, err)
+					fmt.Fprintf(d.ErrOut, "error when evicting pod %q from terminating namespace %q (will retry after 5s): %v\n", activePod.Name, activePod.Namespace, err)
```

Comment:
```suggestion
					fmt.Fprintf(d.ErrOut, ""error when evicting pod %q/%q (will retry after 5s): %v\n", activePod.Namespace, activePod.Name, err)
```

Reply:
We don't think that the fact that the namespace is terminating is interesting?  If someone is investigating the logs for "why is `${POD}` failing to drain?", it seemed like `...terminating namespace "${NAMESPACE}"...` would help them  focus on why that namespace termination was slow, instead of wasting their time poking at the pod specifics.